packages/engine-client/src/types/serverQueryTypes.ts

```diff
@@ -310,11 +309,11 @@ export interface EngineServerQueryResponseByType {
   symbols: EngineServerSymbolsResponse;
   all_products: EngineServerAllProductsResponse;
   health_groups: EngineServerHealthGroupsResponse;
-  order: EngineServerGetOrderResponse;
+  order: EngineServerOrderResponse;
   orders: EngineServerProductOrdersResponse;
   validate_order: EngineServerValidateOrderResponse;
   subaccount_orders: EngineServerSubaccountOrdersResponse;
-  fee_rates: EngineServerSubaccountFeeRatesResponse;
+  fee_rates: EngineServerFeeRatesResponse;
```

Comment:
Keeping the `EngineServer` prefix, but naming is consistent with the name of the backend query (ex. `FeeRates`, `Order`, `HealthGroups`)